gui/interception_filters.py

```diff
@@ -51,7 +51,7 @@ def draw(self):
 
         # Adding some default interception filters
         # self.IFModel.addElement("Scope items only: (Content is not required)") # commented for better first impression.
-        self._extender.IFModel.addElement("URL Not Contains (regex): \\.js|\\.css|\\.png|\\.jpg|\\.svg|\\.jpeg|\\.gif|\\.woff|\\.map|\\.bmp|\\.ico$")
+        self._extender.IFModel.addElement("URL Not Contains (regex): (\\.js|\\.css|\\.png|\\.jpg|\\.svg|\\.jpeg|\\.gif|\\.woff|\\.map|\\.bmp|\\.ico)((\\?(v|ver|version)=)?[^=]+)?$")
```

Comment:
```suggestion
        self._extender.IFModel.addElement("URL Not Contains (regex): \\.(?:js|css|png|jpg|svg|jpeg|gif|woff|map|bmp|ico)(?:\\?(?:v|ver|version)=[^=&]+)?$")
```
`?:` makes a group non-capturing for performance, as the groups aren't referenced.

I think this is intended to match on an appended, optional version URL parameter such as `.js?version=123`), but the current expression also seems to match a URL ending with `.js-arbitrary`.